config/initializers/friendly_id.rb

```diff
@@ -58,7 +58,7 @@
   # separator. If you're upgrading from FriendlyId 4, you may wish to replace this
   # with two dashes.
   #
-  config.sequence_separator = '-'
+  config.sequence_separator = '--'
```

Comment:
This doesn't seem to work with latest friendly-id . It looks like the sample initializer code is outdated. If you try it it will throw

    undefined method `sequence_separator' for #<#<Class: